sdk/tables/azure-data-tables/README.md

```diff
@@ -69,15 +70,19 @@ az storage account keys list -g MyResourceGroup -n MyStorageAccount
 ```
 
 Use the key as the credential parameter to authenticate the client:
+
+<!-- SNIPPET:sample_authentication.auth_from_shared_key -->
+
 ```python
-from azure.core.credentials import AzureNamedKeyCredential
 from azure.data.tables import TableServiceClient
+from azure.core.credentials import AzureNamedKeyCredential
 
-credential = AzureNamedKeyCredential("my_account_name", "my_access_key")
-
-service = TableServiceClient(endpoint="https://<my_account_name>.table.core.windows.net", credential=credential)
+credential = AzureNamedKeyCredential(self.account_name, self.access_key)
+table_service_client = TableServiceClient(endpoint=self.endpoint, credential=credential)
```

Comment:
Having things like `self.endpoint` in the code is really unhelpful for a readme.... even though I understand this is for testing.....

Reply:
Changed back. I don't know if ci has snippet consistency check, if it has, we have to remove the snippet annotations to keep the samples in README clear.